queries.sql

```diff
@@ -135,3 +135,81 @@ SELECT full_name, COUNT(full_name) AS amount FROM owners
 JOIN animals ON animals.owner_id = owners.id
 GROUP BY full_name
 ORDER BY amount DESC LIMIT 1;
+
+
```

Comment:
![image](https://github.com/M-Anwar-Hussaini/Vet-Clinic/assets/93039359/be04500d-548a-44cc-87ab-80a0bee73c16)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you remove the Live Demo section from your readme file since you don't have any working live link in your readme file 😨 Kindly remove this whole Live Demo section to make your readme descriptive and professional 👍